### PR TITLE
refactor: move and rename mock-account-provider -> mock-account-servicing-entity

### LINKED ISSUE #1187: Rename `mock-account-provider` to `mock-account-servicing-entity`

### LINKED ISSUE #1188: Move `infrastructure/local` into its own root `localenv` directory

### LINKED ISSUE #1189: Move `mock-account-servicing-entity` / `mock-account-provider` into `localenv` folder
In order to separate what is Rafiki and what is the local playground

### PR DESCRIPTION
**Pull request by sabineschaller**
## Changes proposed in this pull request

- 

## Context

- fixes #1187 
- fixes #1188 
- fixes #1189

## Checklist

- [x] Related issues linked using `fixes #number`
- [ ] Tests added/updated
- [x] Documentation added
- [ ] Make sure that all checks pass

### CODE REVIEW

File: docs/deployment.md [hunk 1]
```diff
@@ -8,16 +8,16 @@ These packages include
 
 - `backend` (SPSP, Open Payments APIs, Admin APIs, STREAM endpoint)
 - `auth` (GNAP auth server)
-- `mock-account-provider` (mocks an [Account Servicing Entity](./glossary.md#account-servicing-entity))
-- `http-signature-utils` (request signature generation for Postman)
+- `mock-account-servicing-entity` (mocks an [Account Servicing Entity](./glossary.md#account-servicing-entity))
```
Comment (wilsonianb) [inline docs/deployment.md:11]: There's still some "Mock account providers"
https://github.com/interledger/rafiki/blob/main/docs/deployment.md#exploring-accounts-on-mock-account-provider

File: localenv/mock-account-servicing-entity/app/root.tsx
Comment (wilsonianb) [inline localenv/mock-account-servicing-entity/app/root.tsx:1]: https://github.com/interledger/rafiki/blob/478232c90c07e00b385ac9e03b00be8807b34a1e/packages/mock-account-provider/app/root.tsx#L13

File: localenv/cloud-nine-wallet/seed.yml
Comment (wilsonianb) [inline localenv/cloud-nine-wallet/seed.yml:1]: https://github.com/interledger/rafiki/blob/478232c90c07e00b385ac9e03b00be8807b34a1e/infrastructure/local/cloud-nine-wallet/seed.yml#L4

File: localenv/happy-life-bank/seed.yml
Comment (wilsonianb) [inline localenv/happy-life-bank/seed.yml:1]: https://github.com/interledger/rafiki/blob/478232c90c07e00b385ac9e03b00be8807b34a1e/infrastructure/local/happy-life-bank/seed.yml#L4

File: localenv/mock-account-servicing-entity/seed.example.yml
Comment (wilsonianb) [inline localenv/mock-account-servicing-entity/seed.example.yml:1]: https://github.com/interledger/rafiki/blob/478232c90c07e00b385ac9e03b00be8807b34a1e/packages/mock-account-provider/seed.example.yml#L4

File: localenv/mock-account-servicing-entity/app/lib/parse_config.ts
Comment (wilsonianb) [inline localenv/mock-account-servicing-entity/app/lib/parse_config.ts:1]: https://github.com/interledger/rafiki/blob/478232c90c07e00b385ac9e03b00be8807b34a1e/packages/mock-account-provider/app/lib/parse_config.ts#L9

**Review by wilsonianb — COMMENTED** [review]
localenv works for me

**Comment by sabineschaller** [comment]
> There's still some "Mock account providers"

Thank you! I guess I only searched and replaced `mock-account-provider`

File: docs/deployment.md [hunk 2]
```diff
@@ -8,16 +8,16 @@ These packages include
```
Comment (mkurapov) [inline docs/deployment.md:8]: Just above in this file we still have "...mock an account provider"

File: docs/deployment.md [hunk 3]
```diff
@@ -38,17 +38,17 @@ pnpm localenv:stop && pnpm localenv:dbvolumes:remove
...
+![Mock Account Servicing Entity Accounts](./img/map-accounts.png)
 
 The accounts of the second instance (Happy Life Bank) can be found on `localhost:3031`.
 
 When clicking on the Account Name, a list of Transactions appears.
 
-![Mock Account Provider Transactions](./img/map-transactions.png)
+![Mock Account Servicing Entity Transactions](./img/map-transactions.png)
 
 ### Admin APIs
```
Comment (mkurapov) [inline docs/deployment.md:54]: In the Open Payments APIs section, can you please update the eCommerce example section to include the changes I made for splitting out the quote grant creation 🙃

File: docs/deployment.md [hunk 4]
```diff
@@ -8,16 +8,16 @@ These packages include
...
+- `local-http-signatures` (request signature generation for Postman)
 
 These packages depend on the following databases
 
 - TigerBeetle or Postgres (accounting)
 - Postgres (Open Payments resources, auth resources)
 - Redis (STREAM details)
 
-We provide containerized versions of our packages together with two pre-configured docker-compose files ([peer1](../infrastructure/local/docker-compose.yml) and [peer2](../infrastructure/local/peer-docker-compose.yml))to start two Mock Account Providers with their respective Rafiki backend and auth servers. They automatically peer and 2 to 3 user accounts are created on both of them.
+We provide containerized versions of our packages together with two pre-configured docker-compose files ([peer1](../localenv/docker-compose.yml) and [peer2](../localenv/peer-docker-compose.yml))to start two Mock Account Servicing Entities with their respective Rafiki backend and auth servers. They automatically peer and 2 to 3 user accounts are created on both of them.
```
Comment (mkurapov) [inline docs/deployment.md:20]: I think the links 
```suggestion
We provide containerized versions of our packages together with two pre-configured docker-compose files ([peer1](../localenv/cloud-nine-wallet/docker-compose.yml) and [peer2](../localenv/happy-life-bank/peer-docker-compose.yml))to start two Mock Account Servicing Entities with their respective Rafiki backend and auth servers. They automatically peer, and 2 to 3 user accounts are created on both of them.
```

**Review by mkurapov — APPROVED** [review]
localenv works for me :+1:

**Review by mkurapov — APPROVED** [review]

**Pull request merged**